### DISCUSSION TITLE
Where is the "picasso" folder when installing via conda ?

### DISCUSSION THREAD

**Discussion by cleterrier** [jungmannlab/picasso#297] [Q&A]
I followed the conda way of installing picasso (from the readme):

_1. Open the console and create a new conda environment: conda create --name picasso python=3.8
2. Activate the environment: source activate picasso for Linux / Mac Os X or activate picasso for Windows.
3. (Optional) For Mac systems (e.g. M1) install PyQt via conda: conda install -c anaconda pyqt.
4. Install using pip: pip install picassosr.
5. (Optional) If you want to use hdbscan install using pip: pip install hdbscan.
6. (Optional) If you plan to compile your own installer additionally install Pyinstaller: pip install pyinstaller
7. Continue with the installation of Picasso_

First thing, I don't understand how I should "continue with the installation of Picasso" at point 7, as picasso has been installed using pip at point 4. More confusing is that the next steps of the readme actually also detail how to install Picasso (via PyPi or github), but in my understanding these are alternative ways to the conda install, not things you should do after having done steps 1-7 above.

Second thing (main point of this issue), I'm trying to get the Localize module to remember my parameters, so I look into the camera configuration file in the docs (https://picassosr.readthedocs.io/en/latest/localize.html#camera-config):
_Picasso can remember default cameras and will use saved camera parameters. In order to use camera configs, create a file named config.yaml in the picasso folder. To start with a template, modify config_template.yaml that can be found in the folder per default._ 

But I don't know where is the "picasso folder" after installing via conda as above. I have a .picasso folder in my user folder, but it does not contain the picasso files (only a settings.yaml file). Where can I find the config_template.yaml file?

**Comment by rafalkowalewski1** [comment]
Hello Christophe,

Thank you for pointing this out. I will try to make the installation instructions more transparent - they are indeed a bit confusing. You are right, the PyPI and github installations are separate manuals. You can also ignore the 7th point in the instructions you provided in the beginning.

When it comes to cameras configuration, you need to find your picasso environment folder. I assume you're using a mac and you should be able to find it under YOUR_USER_FOLDER/opt/anaconda3/envs/picasso.
Then, you need to find your picassosr package within this environment, which in my case can be found here: YOUR_USER_FOLDER/opt/anaconda3/envs/picasso/lib/python3.8/site-packages/picassosr-0.5.4-py3.8.egg/picasso
This should be the folder when you can create the config.yaml file. Remember that config_template.yaml is ignored by Localize and only config.yaml (that you will create) is processed.

I hope this helped you, please let me know if you encounter any other difficulties : )

Bests,
Rafal

**Reply by cleterrier** [reply]
Hi, I'm on a Windows 10 PC (I use the python install to benefit from GPUFit). What would be the path on it? I can find some Anaconda3 folders but not the rest of the path.

**Reply by cleterrier** [reply]
OK I found it, under Windows it's in
`USER_FOLDER\.conda\envs\picasso\Lib\site-packages\picasso`

**Comment by cleterrier** [comment]
Just an extra question - how to specify the default EM Gain and pixel size in the config.yaml file? I tried to add "EM Gain" and "Pixelsize (nm)" values in the config.yaml file but they are not taken into account.

**Comment by rafalkowalewski1** [comment]
I attach an example file, I hope this helps!

[config.yaml.zip](https://github.com/jungmannlab/picasso/files/10111431/config.yaml.zip)
